content/docs/nav.yml

```
@@ -1,88 +1,88 @@
-- title: Installation
+- title: Instalacja
   items:
     - id: getting-started
-      title: Getting Started
+      title: Podstawowe informacje
     - id: add-react-to-a-website
-      title: Add React to a Website
+      title: Dodaj Reacta do swojej strony
     - id: create-a-new-react-app
-      title: Create a New React App
+      title: Stwórz nową aplikację w Reakcie
     - id: cdn-links
-      title: CDN Links
-- title: Main Concepts
+      title: Linki do CDN-ów
+- title: Podstawowe informacje
   isOrdered: true
   items:
     - id: hello-world
-      title: Hello World
+      title: Witaj, świecie
     - id: introducing-jsx
-      title: Introducing JSX
+      title: Wprowadzenie do JSX
     - id: rendering-elements
-      title: Rendering Elements
+      title: Renderowanie elementów
     - id: components-and-props
-      title: Components and Props
+      title: Komponenty i właściwości
     - id: state-and-lifecycle
-      title: State and Lifecycle
+      title: Stan i cykl życia
     - id: handling-events
-      title: Handling Events
+      title: Obsługa zdarzeń
     - id: conditional-rendering
-      title: Conditional Rendering
+      title: Renderowanie warunkowe
     - id: lists-and-keys
-      title: Lists and Keys
+      title: Listy i klucze
     - id: forms
-      title: Forms
+      title: Formularze
     - id: lifting-state-up
-      title: Lifting State Up
+      title: Wynoszenie stanu w górę
     - id: composition-vs-inheritance
-      title: Composition vs Inheritance
+      title: Kompozycja a dziedziczenie
     - id: thinking-in-react
-      title: Thinking In React
-- title: Advanced Guides
+      title: Myślenie reactowe
+- title: Zaawansowane informacje
   items:
     - id: accessibility
-      title: Accessibility
+      title: Dostępność
     - id: code-splitting
-      title: Code-Splitting
+      title: Dzielenie kodu
     - id: context
       title: Context
     - id: error-boundaries
       title: Error Boundaries
     - id: forwarding-refs
-      title: Forwarding Refs
+      title: Zwracanie Refs
     - id: fragments
       title: Fragments
     - id: higher-order-components
-      title: Higher-Order Components
+      title: Komponenty wyższego rzędu
     - id: integrating-with-other-libraries
-      title: Integrating with Other Libraries
+      title: Integracja z innymi bibliotekami
     - id: jsx-in-depth
-      title: JSX In Depth
+      title: JSX w szczegółach
     - id: optimizing-performance
-      title: Optimizing Performance
+      title: Optymalizacja wydajności
     - id: portals
-      title: Portals
+      title: Portale
     - id: react-without-es6
-      title: React Without ES6
+      title: React bez ES6
     - id: react-without-jsx
-      title: React Without JSX
+      title: React bez JSX
     - id: reconciliation
       title: Reconciliation
```

```suggestion
       title: Rekoncyliacja
```
Z dużym znakiem zapytania.

---

Ciezko mi nawet wypowiedziec to po polsku - wszystkim pasuje?